npm-shrinkwrap.json

```diff
@@ -9211,6 +9211,14 @@
           "version": "1.1.0"
         }
       }
+    },
+    "source-map": {
```

Comment:
:+1: removing the fields isn't a problem unless you happen to be pointing at a github dependency.